docs/spec.md

```diff
@@ -0,0 +1,309 @@
+# BODS Visualiser Specification
+
+This document outlines the functionality and requirements of the [BODS Visualiser](https://github.com/openownership/visualisation-tool).
+
+Refer to the full [BODS documentation](https://standard.openownership.org/en/0.3.0/) for more information (and the associated [github repository](https://github.com/openownership/data-standard)). 
+
+
+
+## Data requirements
+
+The visualiser requires a minimum set of data fields to produce a directed graph. It is not necessary for the supplied JSON data to be valid BODS data, but it must meet these minimum requirements.
+
+This is not a full schema. The following tables represent the opinionated requirements of the visualiser. Any schema fields that are not present in the tables below are not currently used in the generation of graphs by the visualiser tool.
```

Comment:
Should we clarify explicitly that extra fields will be ignored, or is that obvious?